Library/Formula/mono.rb

```diff
@@ -30,7 +34,13 @@ def install
       --prefix=#{prefix}
       --enable-nls=no
     ]
-    args << "--build=" + (MacOS.prefer_64_bit? ? "x86_64": "i686") + "-apple-darwin"
+
+    if build.with? "64bit"
+      args << "--build=x86_64-apple-darwin"
+    else
+      # no option specified
+      args << "--build=" + (MacOS.prefer_64_bit? ? "x86_64": "i686") + "-apple-darwin"
```

Comment:
Because I don't think having a 64-bit option that's only necessary when your OS doesn't support 64-bit binaries makes sense. I also don't think changing an option like this in a minor release makes sense; I'm concerned it'll break other formulae that use `mono` and requires more discussion and/or testing.